gtsam/base/Vector.h

```diff
@@ -25,11 +25,11 @@
 #define MKL_BLAS MKL_DOMAIN_BLAS
 #endif
 
-#include <cmath>
-#include <limits>
 #include <gtsam/global_includes.h>
 #include <Eigen/Core>
+#include <cmath>
```

Comment:
Header bloat! These headers should  go in c++, as mentioned in an earlier comment.

Reply:
These are already present in the cpp file, so just removed them from the header file.